lib/iris/plot.py

```diff
@@ -273,10 +283,11 @@ def _invert_yaxis(v_coord, axes=None):
 
 def _check_contiguity_and_bounds(coord, data, abs_tol=1e-4, transpose=False):
     """
-    Check that the discontinuous bounds occur where the data is masked.
+    Checks that any discontiguity in the bounds of the coordinate only occur
+    where the data is masked.
 
```

Comment:
This is neater and more accurate, but I have made some changes to the name and docstring for this function as well.  Probably worth agreeing on changes.